datastores-parent/datastore-common/src/main/java/org/wicketstuff/datastores/common/SessionQuotaManagingDataStore.java

```diff
@@ -153,10 +153,10 @@ synchronized void removeAllPages(IPageContext context, IPageStore delegate) {
 		/**
 		 * Removes a page by its identifier
 		 *
-		 * @param pageId The id of the page to remove
-		 * @return 
```

Comment:
`@return` is necessary here to make javadoc happy :)

Reply:
@theigl could you please address this one? :)

Reply:
This is already addressed. For some reason this was not marked as outdated by GitHub.